pkgs/development/python-modules/pgvector/default.nix

```
@@ -16,7 +16,7 @@
 
 buildPythonPackage rec {
   pname = "pgvector";
-  version = "0.1.8";
+  version = "0.2.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.6";
```

```suggestion
  disabled = pythonOlder "3.8";
```

Doesn't matter much, but as the changelog explicitly mentions it.

---

Thanks! updated it.